doc/steps.md

```diff
@@ -571,7 +573,23 @@ Output facts:
 
 ### Step `backup_stop`
 
-Stop started [backup](/doc/backup.md) on the instance.
+Stop started [backup](/doc/backups.md) on the instance.
+
+### Step `restore`
+
+[Restore](/doc/backups.md#how-to-restore-from-saved-backup) each instance from a [backup](#step-backup) archive.
+
+Input variables from config:
+
+- `cartridge_paths_to_keep_before_restore` - list of full paths or relative paths
+  to work/memtx/vinyl/wal directory that should be kept on instance restore
+  (`.tarantool.cookie` will be kept independently of this variable);
+  it's possible to use bash patterns, e.g. `*.control`.
+- `cartridge_restore_backup_path` - path to the instance backup archive on the remote machine;
+- `cartridge_remote_backups_dir` - directory with backups on the remote;
+- `cartridge_force_restore` - flag indicates that conflicting files should be overwritten;
```

Comment:
BTW, what if somebody wants to restore all files except some special? I mean, not rewrite some files from backup?

Reply:
I think it's too difficult, and not necessary.